conf/app.routes

```diff
@@ -6,3 +6,5 @@ DELETE  /income-tax/nino/:nino/sources/:submissionId        controllers.DeleteCI
 
 POST    /income-tax/nino/:nino/sources                      controllers.CreateUpdateCisDeductionsController.postCISDeductions(nino: String, taxYear: Int)
 
+GET     /:taxYear/tasks/:nino                               controllers.CommonTaskListController.getCommonTaskList(taxYear: Int, nino: String)
```

Comment:
Should we follow the same format as the other routes?
Maybe `GET /income-tax/nino/:nino/tasks controllers.CommonTaskListController.getCommonTaskList(taxYear: Int, nino: String)`